docs/apiclient.rst

```diff
@@ -18,4 +20,60 @@ To initialize the APIClient, you need to provide the hostname and port of the FE
    from fedn import APIClient
    client = APIClient("localhost", 8092)
 
+Inital setup
+------------
+
+Before you can start training models, you need to set the active package and an initail seed model. The active package can be set using the following code snippet:
+
+.. code-block:: python
+   
+   client.set_active_package(path="path/to/compute_package.zip", helper="numpyhelper")
+
+To set the initial seed model, you can use the following code snippet:
+
+.. code-block:: python
+   
+   client.set_active_model(path="path/to/seed_model.zip")
```

Comment:
They are not usually zipped. Usually ends with ".npz" (numpy compressed)

Reply:
changed